### PR TITLE
Keras cleanup

### PR DESCRIPTION
**Pull request by gogasca**
- Remove parameters in optimizer which were causing training not to grow. Just kept learning rate.
- Refactor Keras layers in model
- Replace Hparas with args

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/404)

### CODE REVIEW

File: census/tf-keras/trainer/model.py [hunk 1]
```diff
@@ -54,31 +54,29 @@ def input_fn(features, labels, shuffle, num_epochs, batch_size):
...
-          activation=tf.nn.relu,
-          kernel_initializer='uniform',
-          input_shape=(input_dim,)))
-  model.add(tf.keras.layers.Dense(75, activation=tf.nn.relu))
-  model.add(tf.keras.layers.Dense(50, activation=tf.nn.relu))
-  model.add(tf.keras.layers.Dense(25, activation=tf.nn.relu))
-  # The single output node and Sigmoid activation makes this a Logistic
-  # Regression.
-  model.add(tf.keras.layers.Dense(1, activation=tf.nn.sigmoid))
+  l = tf.keras.layers
```
Comment (alecglassford) [inline census/tf-keras/trainer/model.py]: Nit: The point of this is just to save characters in the next few lines, right? I don't love the [single character `l` variable name](http://google.github.io/styleguide/pyguide.html#3161-names-to-avoid). Would `Dense = tf.keras.layers.Dense` work just as well? (and then just use `Dense` instead of `l.Dense`).

File: notebooks/tensorflow/getting-started-keras.ipynb [hunk 2]
```diff
@@ -3814,11 +3810,16 @@
...
         "# Setup Learning Rate decay.\n",
-        "lr_decay = tf.keras.callbacks.LearningRateScheduler(\n",
+        "lr_decay_cb = tf.keras.callbacks.LearningRateScheduler(\n",
         "    lambda epoch: LEARNING_RATE + 0.02 * (0.5 ** (1 + epoch)),\n",
-        "    verbose=True)"
+        "    verbose=True)\n",
+        "\n",
+        "# Setup TensorBoard callback.\n",
+        "tensorboard_cb = tf.keras.callbacks.TensorBoard(\n",
+        "      os.path.join(args.job_dir, 'keras_tensorboard'),\n",
```
Comment (alecglassford) [inline notebooks/tensorflow/getting-started-keras.ipynb]: I don't think this works, because when running in the notebook the `args` variable is not actually defined (since we're not parsing command-line flags).

You could either export locally (`keras_tensorboard`) or to the Cloud Storage dir that the user has already set up (`JOB_DIR + '/keras_tensorboard'`) - compare to both options in the  ["Export the model for serving" section](https://colab.sandbox.google.com/github/GoogleCloudPlatform/cloudml-samples/blob/master/notebooks/tensorflow/getting-started-keras.ipynb#scrollTo=gmTiOQObmTqd). Depends on how you want to use TensorBoard (see overall comment).

File: census/tf-keras/trainer/model.py (hunk 1 above)
Comment (gogasca) [reply census/tf-keras/trainer/model.py]: Thanks, updated. I think this has become a bad standard in TF-Keras

**Comment by gogasca** [comment]
@alecglassford PTAL thanks for the review

**Comment by gogasca** [comment]
Thanks Alec
